From bb0678b1a92540d0b3764c6577c67edf505e1b8a Mon Sep 17 00:00:00 2001
From: Radha Mohan Chintakuntla <radhac@marvell.com>
Date: Tue, 21 May 2019 20:25:57 -0700
Subject: [PATCH 247/386] octeontx: Update the SMC call macros for GPIO
 interrupts

ATF has updated the SMC call macros to enable/disable GPIO interrupts.
Thi patch syncs the macros in the kernel drivers accordingly.

Change-Id: Ida35195399f4f264a6b203b3b845c9cafe2f7e0b
Signed-off-by: Radha Mohan Chintakuntla <radhac@marvell.com>
Reviewed-on: https://sj1git1.cavium.com/9414
Tested-by: sa_ip-sw-jenkins <sa_ip-sw-jenkins@marvell.com>
Reviewed-by: Sunil Kovvuri Goutham <Sunil.Goutham@cavium.com>
[RH: Original patch taken from marvell 88F3720 board support SDK 10.0-PR2003]
Signed-off-by: Ruiqiang Hao <Ruiqiang.Hao@windriver.com>
---
 drivers/net/ethernet/cavium/octeontx-83xx/octeontx.h | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/net/ethernet/cavium/octeontx-83xx/octeontx.h b/drivers/net/ethernet/cavium/octeontx-83xx/octeontx.h
index d2d0f3473705..6305618e6c14 100644
--- a/drivers/net/ethernet/cavium/octeontx-83xx/octeontx.h
+++ b/drivers/net/ethernet/cavium/octeontx-83xx/octeontx.h
@@ -52,9 +52,9 @@ struct wqe_s {
 
 /* THUNDERX SMC definitons */
 /* X1 - gpio_num, X2 - sp, X3 - cpu, X4 - ttbr0 */
-#define THUNDERX_INSTALL_GPIO_INT       0x43000801
+#define THUNDERX_INSTALL_GPIO_INT       0xc2000801
 /* X1 - gpio_num */
-#define THUNDERX_REMOVE_GPIO_INT        0x43000802
+#define THUNDERX_REMOVE_GPIO_INT        0xc2000802
 
 struct intr_hand {
 	u64	mask;
-- 
2.17.1

